fstab.yaml

```diff
@@ -1,2 +1,2 @@
 mountpoints:
-  /: https://adobe.sharepoint.com/:f:/r/sites/HelixProjects/Shared%20Documents/sites/tiaa
+  /: https://adobe.sharepoint.com/:f:/r/sites/HelixProjects/Shared%20Documents/sites/tiaa/drafts/itavy
```

Comment:
The mountpoint should remain the same, you don't have to edit it to use the drafts.

Reply:
removed